throubleshoot/no-secret.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: no-secret
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: no-secret
+  template:
+    metadata:
+      labels:
+        app: no-secret
```

Comment:
I'd call it `secret-missing` instead of `no-secret`, same for the other `no-*` ones